bitnami/vault/values.yaml

```diff
@@ -368,6 +368,10 @@ server:
   ## @param server.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the Vault Server container(s)
   ##
   extraVolumeMounts: []
+  ## e.g. for oracle plugin installation:
```

Comment:
Please avoid references to third party companies in the documentation.

Reply:
I have updated